experiments/deploying_code.md

```diff
@@ -1,12 +1,12 @@
 # How You Get Code To 100% Of Release
 
-_Warning: this is written by NOT a Release Manager, and has not (yet) been vetted.  Reflects state of the world on 2018/07/19._
+_Warning_: this is written by NOT a Release Manager, and has not (yet) been vetted.  Reflects state of the world on 2018/07/19.
```

Comment:
Feature roll-out seems outside of our team's wheelhouse? I argue this document is better to be written and maintained by Benson's team.